client/dashboard/app/queries/domain.ts

```diff
@@ -1,8 +1,18 @@
-import { queryOptions } from '@tanstack/react-query';
-import { fetchDomain } from '../../data/domain';
+import { queryOptions, mutationOptions } from '@tanstack/react-query';
+import { fetchDomain, disconnectDomain } from '../../data/domain';
+import { queryClient } from '../query-client';
 
 export const domainQuery = ( domainName: string ) =>
 	queryOptions( {
 		queryKey: [ 'domains', domainName ],
 		queryFn: () => fetchDomain( domainName ),
 	} );
+
+export const disconnectDomainMutation = ( domainName: string ) =>
+	mutationOptions( {
+		mutationKey: [ 'domains', domainName, 'disconnect' ],
```

Comment:
I don't think this key is being used anywhere 😄

Reply:
Removed! 👍